ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -994,140 +994,46 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: rosa-aws-sts-shared-vpc
-- as: aws-rosa-sts-hcp-full-f14
-  cron: 53 4 14,29 * *
+- as: aws-rosa-sts-hcp-int-full-f7
+  cron: 1 21 7,14,21,28 * *
   steps:
     cluster_profile: aws-sd-qe
     env:
       CHANNEL_GROUP: nightly
       E2E_RUN_TAGS: '@amd64 and @aws-ipi and @rosa and @hypershift-hosted'
-      OPENSHIFT_VERSION: "4.14"
-      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;~NonHyperShiftHOST&;ROSA&
-    test:
-    - chain: openshift-e2e-test-hypershift-qe
-    workflow: rosa-aws-sts-hypershift
-- as: aws-rosa-sts-hcp-advanced-full-f14
```

Comment:
Let us keep the same job list as 4.15. On the stable.yaml, we only do the cluster health checking, while do the OCP e2e testing with the nightly builds to cover the EC/RC and the Z stream openshift version.

Reply:
These jobs are not redundant jobs, we need to keep them here.

Reply:
> Let us keep the same job list as 4.15

Yes, I just copied the 4.15 jobs to 4.14. There were too many jobs before, so we deleted some jobs